fullscreen-custom/definition.json

```diff
@@ -57,7 +57,11 @@
             "fr": "Image"
           },
           "type": "image",
-          "default": "./assets/images/sample-image-1.jpg"
+          "default": "./assets/images/sample-image-1.jpg",
+          "tooltip": {
+            "en": "dimension: 1024 × 1024",
+            "fr": "dimension: 1024 × 1024"
```

Comment:
Why did we set these arbitrary values? I think we should come up with a valid set of dimensions for each template.
Also, I think this tooltip deserves a better description, such as: `Recommended image dimensions: <width> x <height>`

Reply:
it is necessary to add a tooltips for the requests "browse files" ?

Reply:
Not sure what you mean here, can you explain it some more please?

Reply:
![Capture d’écran 2020-11-02 à 14 03 50](https://user-images.githubusercontent.com/48413069/97871107-450d8f00-1d14-11eb-8dc3-b4ebbb639e48.png)
for this, if there is no basic image no need for tooltips ? (it sounds obvious but I wanted to know)

Reply:
The tooltip should always be there, even if there is no image by default. But the sizes recommended will vary. I think this part needs some discussion, because the sizes can and WILL change for each template

Reply:
Ok, I change all files composed of `"type": "image"`